### DISCUSSION TITLE
Add support for Scheduled Events

### DISCUSSION THREAD

**Discussion by Nimja** [discordjs/discord.js#7105] [Ideas]
This is a feature for community servers: https://support.discord.com/hc/en-us/articles/4409494125719-Scheduled-Events

It is supported in the API: https://discord.com/developers/docs/resources/guild-scheduled-event

Worth adding?

**Comment by Jiralite** [comment]
See https://github.com/discordjs/discord.js/pull/6493.

**Reply by dahool** [reply]
has this been implemented? I see the PR has been merged, but there's nothing on the documentation on how to use it or how to create an event.

**Reply by Jiralite** [reply]
This was released over a year ago, in a version that is not even the latest major. The documentation for the current stable release is here:

https://discord.js.org/#/docs/discord.js/14.8.0/class/GuildScheduledEventManager
